api/openapi-spec/swagger.json

```diff
@@ -9242,6 +9242,12 @@
    },
    "v1.CustomizeComponentsPatch": {
     "type": "object",
+    "required": [
+     "resourceName",
+     "resourceType",
+     "patch",
+     "type"
+    ],
```

Comment:
Is this related to this PR? let's just keep only those changes that are relevant to this PR.